docs/pages/guides/sell-nft.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Selling an NFT'
+---
+
+import { Steps } from 'nextra-theme-docs'
+
+# Selling an NFT
+
+## Introduction
+
+You can create a listing on any NFTs you own directly on the Liteflow dashboard.
+
+## Prerequisites
+
+Before creating an listing, ensure you have:
+
+- Imported or created the collection using the Liteflow dashboard
+- [Set Up the currency you want to use](./setup-currencies).
+- You are owner of the NFT
+
+## Steps
+
+<Steps>
+
+### Access Listings section
+
+Navigate to the [Listings](https://dashboard.liteflow.com/listings) section in your Liteflow dashboard.
+
+### Start creating a listing
+
+Click on the 'Create' button to open the form to create a listing on an NFT.
+
+### Select an NFT
+
+Choose the NFT from the list.
+
+If the NFT is not listed but is listed in the [NFTs](https://dashboard.liteflow.com/nfts) section, it means your wallet is not the owner of this NFT. Make sure your are login with the right wallet.
+
+### Optional - switch network
+
+If the NFT is on a different network than your current wallet is, you need to switch to the same network as the NFT. Open your wallet and select the right network. You will need to login again and come back to this page.
+
+### Set the listing details
+
+- **Currency**: Select the currency you want to use.
+- **Price per unit**: Enter the price you want to sell the NFT for.
+- **Expiration date**: Select when the listing will expire.
+
+### Create the listing
+
+Press the "Create" button and follow the steps in your wallet.
+
+Once the listing is successfully created, you will be redirected to its detail page.
```

Comment:
It would be great to add the part to generate the link or redirect to the starter kit because as it is right now I created something but I still don't know how to sell it

Reply:
I was wondering about it but because the admin PR is not merged yet I didn't add the generate link explanation.
I already added a todo on https://github.com/liteflow-labs/admin/pull/409.

But yes I will add something about the listing on the starter-kit.

Reply:
added

Reply:
payment link instruction added with https://github.com/liteflow-labs/liteflow-js/pull/230/commits/44bc46227e4b25aea065c31eb5a93f79cce51d72